django_project/localities_osm/serializer/locality_osm.py

```diff
@@ -42,3 +44,28 @@ class Meta:
         fields = ['row', 'osm_id', 'osm_type',
                   'type', 'name', 'changeset_version',
                   'changeset_timestamp', 'changeset_user']
+
+
+class LocalityOSMUpdates(ModelSerializer):
+
+    class Meta:
+        model = LocalityOSMView
+        fields = [
+            'pk',
+            'osm_id',
+            'name',
+            'changeset_id',
+            'changeset_version',
+            'changeset_timestamp',
+            'changeset_user',
+        ]
+
+    def to_representation(self, instance):
+        result = super(LocalityOSMUpdates, self).to_representation(instance)
+        result['changeset__social_user__username'] = None
```

Comment:
could you use field serializer so we have nice code?

Reply:
ok @meomancer